.rubocop.yml

```diff
@@ -30,6 +30,7 @@ AllCops:
   UseCache: true
   DisabledByDefault: true
   SuggestExtensions: false
+  NewCops: enable
```

Comment:
I don't feel adamantly about it, but disagree 😬

We opt-in to new ones for base rubocop, and I'd rather opt in to them here too. Some of them are good, and we can enable/disable as we like.

Reply:
sounds good, I'll flip it

Reply:
updated in 9e343c7cc853e823d0e9393898cd485f40c9662d

Reply:
Would this also prevent the warning message from being logged when there are new rules available? Would we want that?

Reply:
I guess I could also remove this line, so it continues to warn, so we get reminded every time there are new cops?

Reply:
> I guess I could also remove this line, so it continues to warn, so we get reminded every time there are new cops?

Small preference from me to keep the warnings, since they're good motivation to act 😄

Reply:
Yep, agree!

Reply:
Done: 7c30ab9ae2d5cb0cd62f73c6641d8571318a303a